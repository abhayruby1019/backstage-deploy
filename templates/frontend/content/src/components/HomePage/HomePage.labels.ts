import Labels, { getLabelByLocale } from '@/utils/labels'

const labels: Labels = {
  default: {
    title: 'Poonawala Fincop Ltd - demo',
  },
}

export default (locale: string) => getLabelByLocale(labels, locale)
