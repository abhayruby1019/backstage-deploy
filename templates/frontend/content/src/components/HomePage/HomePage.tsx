'use client'

import pageLabels from './HomePage.labels'

export default function HomePage() {
  const labels = pageLabels('')
  return <h1>{labels.title}</h1>
}
