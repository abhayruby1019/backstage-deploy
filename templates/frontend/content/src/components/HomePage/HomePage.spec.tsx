import { render } from '@testing-library/react'
import HomePage from './HomePage'

it('renders homepage', () => {
  const { container } = render(<HomePage />)
  expect(container).toMatchSnapshot()
})
