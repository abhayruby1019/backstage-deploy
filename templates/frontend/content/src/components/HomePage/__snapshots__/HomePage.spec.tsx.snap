// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage 1`] = `
<div>
  <h1>
    Poonawala Fincop Ltd - demo
  </h1>
</div>
`;
